src/pages/policyPage.js

```diff
@@ -103,7 +102,7 @@ export default function PolicyPage({ getTenants, tenantValues, thisTenant, graph
     let datAccumulator = [];
     for (const service of servicesResponse) {
       axios
-        .get((typeof env !== 'undefined' ? env.ANUBIS_API_URL : '') + 'v1/policies' + queryParameters, {
+        .get((typeof env !== 'undefined' ? env.ANUBIS_API_URL : '') + 'v1/policies' + '/?limit=-1' + queryParameters, {
```

Comment:
the real solution to the issue is loading data in a ui page on page number change by issuing the proper call to the api. so this does not fix the issue per se, it is a work around.

1) don't close the related issue
2) comment in the code so that we keep track of this being a work around